radio/src/storage/storage_common.cpp

```diff
@@ -98,6 +98,11 @@ void postRadioSettingsLoad()
     g_eeGeneral.internalModule = DEFAULT_INTERNAL_MODULE;
   }
 #endif
+#if defined(STICK_DEAD_ZONE)
+  if (!g_eeGeneral.stickDeadZone) {
+    g_eeGeneral.stickDeadZone = DEFAULT_STICK_DEADZONE;
+  }
+#endif
```

Comment:
While this *does* cause deadzone to be turned back on back to "4" when set to "0", I would prefer to have it as it is now, and remove/improve this in the next version, than require all users to go in and set it. The amount of deadzone at stick center is minor, and this was the _default_ setting for the OTX fork people were using with the NV14.

Reply:
Happens to me too, the settings keep reverting from 0 ot 4 on reboot of the NV14. I actually like the hypersensitivity of the sticks so I have to go back and re-set them to 0 every time I boot the radio.

I was looking for a LUA command to set the deadzone on reboot but noticed the setting only got implemented in the latest version in the first place, so maybe in a later release?

Reply:
In a later release, yet... not all settings can be changed by lua. This and ADC jitter settings are in the back of my mind to add though, particular the latter, as it can then be used in the model wizard to turn jitter filter off for multirotor.

Reply:
Sounds great, looking forward to it. I opened an issue about this (did not see your reply yet) adding the 'internal RF' setting to the 'pile' ;)